### PR TITLE
Feature/new ci

### PR DESCRIPTION
**Pull request by memsharded**
Should be low risk, while enabling/making easier CI with new bintray. Maybe a minor?

### CODE REVIEW

**Comment by memsharded** [comment]
Related to https://github.com/conan-io/conan-package-tools/pull/75

File: conans/client/new_ci.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  - chmod +x .travis/run.sh
+  - ./.travis/run.sh
+"""
+
+linux_config = """      - os: linux
+        services:
+          - docker
+        sudo: required
+        language: python
+        env: CONAN_GCC_VERSIONS={version} CONAN_CURRENT_PAGE=gcc_{name} CONAN_USE_DOCKER=1
```
Comment (lasote) [inline conans/client/new_ci.py]: Cleaner to specify the docker image, I'm working to see if everything fits

File: conans/client/new_ci.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        env: CONAN_CURRENT_PAGE=apple-clang_{version}
+"""
+
+build_py = """from conan.packager import ConanMultiPackager
+
+
+if __name__ == "__main__":
+    builder = ConanMultiPackager(username="{user}", channel="{channel}",
+                                 visual_versions={visual_versions},
+                                 gcc_versions={linux_gcc_versions},
```
Comment (lasote) [inline conans/client/new_ci.py]: We will add clang_versions too, pending some work in package tools

File: conans/client/new_ci.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    fi
+
+    pyenv install 2.7.10
+    pyenv virtualenv 2.7.10 conan
+    pyenv rehash
+    pyenv activate conan
+fi
+
+pip install conan_package_tools # It install conan too
+pip install conan --upgrade
```
Comment (lasote) [inline conans/client/new_ci.py]: Installing first conan relieves the requirements constraints.

File: conans/client/new_ci.py [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+  - pip.exe install conan_package_tools # It install conan too
+  - pip.exe install conan --upgrade
+  - conan user # It creates the conan data directory
+
+test_script:
+  - python build.py
+"""
+
+
+def get_build_py(name, user, channel, visual_versions, linux_gcc_versions, osx_clang_versions,
```
Comment (lasote) [inline conans/client/new_ci.py]: clang_versions

File: conans/client/command.py [hunk 5]
```diff
@@ -72,10 +72,24 @@ def new(self, *args):
         parser.add_argument("-b", "--bare", action='store_true', default=False,
                             help='Create the minimum package recipe, without build() or package()'
                             'methods. Useful in combination with "package_files" command')
+        parser.add_argument("-cis", "--ci_shared", action='store_true', default=False,
```
Comment (lasote) [inline conans/client/command.py]: Shouldn't be defaulted to True? I don't want to write

-cis -cil -cio -ciw -ciu  in a new command

**Comment by lasote** [comment]
Updated!

**Pull request merged**